src/pages/SafeAreaInsetProps.ts

```diff
@@ -0,0 +1,8 @@
+type SafeAreaInsetProps = {
```

Comment:
This type isn't necessary, use `EdgeInsets` instead. Also this file is located in pages for some reason, remove it please